src/api-service/__app__/onefuzzlib/proxy.py

```diff
@@ -90,6 +92,25 @@ def init(self) -> None:
                 self.save_proxy_config()
                 self.set_state(VmState.extensions_launch)
         else:
+            nsg = NSG(
+                name=self.region,
+                region=self.region,
+            )
+
+            result = nsg.create()
+            if isinstance(result, Error):
+                self.set_failed(result)
+                return
+
+            nsg_config = NetworkSecurityGroupConfig()
+            nsg_config.allowed_ips = ["*"]
```

Comment:
Change allowed_ips from ["*"]  to the value set in InstanceConfig

Reply:
Changed.